Library/Formula/sngrep.rb

```diff
@@ -0,0 +1,23 @@
+class Sngrep < Formula
+  homepage "https://github.com/irontec/sngrep"
+  url "https://github.com/irontec/sngrep/archive/v0.2.2.tar.gz"
+  sha1 "c73a6bbc92f2d6a729566af5179fd550a8f74dbd"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "openssl"
+
+  def install
+    system "./bootstrap.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--with-openssl=#{Formula["openssl"].opt_prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/sngrep", "-v"
```

Comment:
Not reasonably, I think, though I'm open to suggestions - a) it needs to be run as either root to capture packets, or run from a pcap file which I would need to include somehow and b) in either case it's a ncurses ui, with no non-interactive mode so I'm not aware of a practical way to actually test on that.